adrs/3610135900/README.md

```diff
@@ -0,0 +1,64 @@
+# Secret Value Syntax
+
+- **State:** Draft
+- **Created:** 2023-08-13
+- **Tags:** secret
+
+## Context
+
+The current ecosystem lacks a standardized format for secret values. This
+decentralization of secret format decisions leads to a situation where
+developers often define the format themselves. Such a disparate approach results
+in many developers either forgetting to set up scanning tools or being unable to
+utilize preconfigured scanning tools.
+
+The lack of a consistent secret format introduces vulnerabilities and potential
+security risks. The changes in GitHub Authentication Token Formats, among other
+companies adding more meaningful prefixes to their secrets, highlight the need
+for a more standardized approach.
+
+### Decision Drivers:
+
+**Security:** A standard format would enable more effective use of secret
+scanning tools, reducing the chances of leaks or unauthorized access.
+
+**Consistency:** A consistent secret format can streamline processes and reduce
+the chance of human error.
+
+**Usability:** Developers can more easily take advantage of preconfigured
+scanning tools with a standardized format.
+
+## Resolution
+
+The agreed-upon structure will be similar to the URN syntax:
+
+```text
+SECRET = "secret:[NID]:[NSS]:[secret value]
+```
+
+- "secret:" is the required case-sensitive scheme at the beginning.
+- NID (Namespace Identifier) defines the specific context or namespace.
+  It isa optional case-sensitive and consist of only alphanumeric characters and
+  dashes value. Regex: `[a-z0-9-]`.
+- NSS (Namespace-Specific String) is unique within the specific namespace.
+  It is a optional case-sensitive and consist of only alphanumeric characters
+  and dashes value. Regex: `[a-z0-9-]`.
+
+The secret regex value is:
+
+```regexp
+^secret:(?:[a-z0-9-]*):(?:[a-z0-9-]*):(.+)$
+```
+
+For example, a Personal Access Token for GitHub could be in the format:
+
+```text
+secret:github:ghp:[.... secret value ....]
```

Comment:
Suggestion: use an example that shows the NSS is not a substitute for the `ghp_` prefix GitHub uses

```suggestion
secret:github:ghp:ghp_000000000000000000000000000000000000
```